src/service/viewport/viewport-impl.js

```diff
@@ -875,7 +886,13 @@ export class ViewportImpl {
     this.lastPaddingTop_ = this.paddingTop_;
     this.paddingTop_ = paddingTop;
 
-    const animPromise = this.animateFixedElements_(duration, curve, transient);
+    const animPromise = this.fixedLayer_.animateFixedElements(
```

Comment:
There was a small bug here before. We need to test that `if (fixedLayer_)` since it's declared as nullable.

Reply:
Done.